python/tests/integration/completions/e2e_text_completion.py

```diff
@@ -79,7 +80,7 @@ async def summarize_function_test(kernel: sk.Kernel):
     )
     output = str(summary).strip()
     print(f"Summary using input context: '{output}'")
-    assert "humans" in output or "Humans" in output or "preserve" in output
+    assert "human" in output or "Human" in output or "preserve" in output
```

Comment:
I used the plural because "human" is in the input and there is a risk of the test passing if the function returns the input instead of a summary. Any reason for this change?

Reply:
That makes sense. I was getting responses like 
"Summary using context and additional string: 'Protect humans, obey, self-preserve'
Summary using context, additional variables, and additional string: 'Human safety, obey, self-preservation, birds.'"